multi-agent/lib/mas/core/auth/credentials/models.py

```
@@ -28,6 +28,12 @@ class TokenStatus(str, Enum):
     REVOKED = "revoked"
 
 
+class StaticAuthMethod(str, Enum):
+    """Built-in auth dropdown options. Registry servers use free-form identifiers."""
+    NONE = "none"
+    ACCESS_TOKEN = "access_token"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "ACCESS_TOKEN"

(S105)